pkg/users/users.js

```diff
@@ -153,15 +161,33 @@ function AccountsPage() {
     } else if (path.length === 1) {
         return (
             <AccountDetails accounts={accountsInfo} groups={groupsExtraInfo}
-                            current_user={current_user_info?.name} user={path[0]} shells={shells} />
+                current_user={current_user_info?.name} user={path[0]} shells={shells} />
         );
     } else return null;
 }
 
-async function getLogins() {
+async function getLastlogPath() {
+    try {
+        await cockpit.spawn(["test", "-e", "/var/run/utmp"], { err: "ignore" });
+        return "/var/run/utmp";
+    } catch (err1) {
+        try {
+            await cockpit.spawn(["test", "-e", "/var/lib/lastlog/lastlog2.db"], { err: "ignore" });
+            return "/var/lib/lastlog/lastlog2.db";
+        } catch (err2) {
+            return null;
+        }
+    }
+}
+
+async function getLogins(LastLogVersion) {
```

Comment:
What about passing the `lastLogPath` here instead of LastLogVersion, and the use code like

```
if (LastLogPath == "/var/lib/lastlog/lastlog2.db")
  lastlog = await cockpit.spawn(["lastlog2"], { environ: ["LC_ALL=C"] });
else
 ...
```

This doesn't require the caller of this function to know how to determine the version.

Also, we don't have to mention utmp this way. The connection between lastlog and utmp doesn't seem to be strong to me, tbh.